docs/ECLLanguageReference/ECLR_mods/BltInFunc-SEQUENTIAL.xml

```diff
@@ -40,52 +40,27 @@
   <para>The <emphasis role="bold">SEQUENTIAL </emphasis>action executes the
   items in the <emphasis>actionlist</emphasis> in the order in which they
   appear in the <emphasis>actionlist</emphasis>. This is useful when a
-  subsequent action requires the output of a precedent action.</para>
+  subsequent action requires the output of a precedent action. By definition,
+  PERSIST on an definition means the definition is evaluated outside of any
```

Comment:
This isn't strictly true.  The persist (or any other workflow items) are evaluated before the first sequential action that requires it.  So SEQUENTIAL may effect the order in which persists are evaluated, but each persist will only be executed once. It may be clearer not to try and describe the ordering.